src/runtime_src/core/common/api/xrt_module.cpp

```diff
@@ -389,11 +400,16 @@ class module_impl
 class module_elf : public module_impl
 {
   xrt::elf m_elf;
-  uint8_t m_os_abi;
+  uint8_t m_os_abi = Elf_Amd_Aie2p;
```

Comment:
we initialize m_os_abi in constructor, then we done need this or there are chance it can be null ?

Reply:
m_os_abi is initialized here because all other member variables like m_ctrl_packet_exist are initialized in class.
I cannot leave one uninitialized